src/python_routing_v02/compute_nhd_routing_SingleSeg_v02.py

```diff
@@ -396,9 +437,9 @@ def main():
                         nts,
                         reach_list,
                         independent_networks[tw],
-                        param_df_sub.index.values,
-                        param_df_sub.columns.values,
-                        param_df_sub.values,
+                        channel_initial_states_df.index.values,
+                        channel_initial_states_df.columns.values,
+                        channel_initial_states_df.values,
```

Comment:
this should still ref param_df_sub

Reply:
at line 434, 
`
q0_sub = channel_initial_states_df.loc[r].sort_index()
`

Reply:
@jhrehanoaa -- rebased to latest upstream/master